gluoncv/model_zoo/efficientnet.py

```diff
@@ -0,0 +1,508 @@
+                                            block_arg.stride,
+                                            block_arg.se_ratio,
+                                            global_params.drop_connect_rate)
+                                     )
+                    if block_arg.num_repeat > 1:
+                        block_arg = block_arg._replace(
+                            input_filters=block_arg.output_filters, stride=1)
+                    for _ in range(block_arg.num_repeat - 1):
+                        self._blocks.add(
+                            MBConv(
+                                block_arg.input_filters,
+                                block_arg.output_filters,
+                                block_arg.expand_ratio,
+                                block_arg.kernel_size,
+                                block_arg.stride,
+                                block_arg.se_ratio,
+                                global_params.drop_connect_rate))
+
+            # Head
+            out_channels = round_filters(1280, self._global_params)
+            self._conv_head = nn.HybridSequential(prefix='conv_head_')
+            with self._conv_head.name_scope():
+                _add_conv(
+                    self._conv_head,
+                    out_channels,
+                    active=True,
+                    batchnorm=True)
+            # Final linear layer
+            self._dropout = self._global_params.dropout_rate
+            self._fc = nn.Dense(self._global_params.num_classes)
+
+    def hybrid_forward(self, F, x, training):
+        x = self.features(x)
+        for block in self._blocks:
+            block.get_mode(training=training)
+            x = block(x)
+        x = self._conv_head(x)
+        x = F.squeeze(
+            F.squeeze(
+                mx.nd.contrib.AdaptiveAvgPooling2D(
+                    x,
+                    1),
+                axis=-1),
+            axis=-1)
+        if self._dropout:
+            x = F.Dropout(x, self._dropout)
+        x = self._fc(x)
+        return x
+
+
+def efficientnet(model_name, return_input_resolution=True):
+    params_dict = {  # (width_coefficient, depth_coefficient, input_resolution, dropout_rate)
+        'efficientnet-b0': (1.0, 1.0, 224, 0.2),
+        'efficientnet-b1': (1.0, 1.1, 240, 0.2),
+        'efficientnet-b2': (1.1, 1.2, 260, 0.3),
+        'efficientnet-b3': (1.2, 1.4, 300, 0.3),
+        'efficientnet-b4': (1.4, 1.8, 380, 0.4),
+        'efficientnet-b5': (1.6, 2.2, 456, 0.4),
+        'efficientnet-b6': (1.8, 2.6, 528, 0.5),
+        'efficientnet-b7': (2.0, 3.1, 600, 0.5)
+    }
+    width_coeff, depth_coeff, input_resolution, dropout_rate = params_dict[model_name]
+    blocks_args, global_params = efficientnet_param(width_coeff, depth_coeff, dropout_rate)
+    model = EfficientNet(blocks_args, global_params)
+    if return_input_resolution:
+        return model, input_resolution
+    return model
+
+
+def efficientnet_b0(pretrained=False):
```

Comment:
At least, users need to be able to customize the number of classes.

Reply:
And also `ctx`. Please use the style as https://github.com/dmlc/gluon-cv/blob/master/gluoncv/model_zoo/resnetv1b.py#L340